### ISSUE TITLE
How can we get bloggers to contribute content back to OTel docs/readme?

### ISSUE THREAD

**Issue by reese-lee** [open-telemetry/sig-end-user#48]

**Comment by avillela** [comment]
Related PR: https://github.com/open-telemetry/community/pull/2162

**Comment by danielgblanco** [comment]
Are we talking about end-user bloggers, that use OTel within their stack? Or maintainers? If it's the former, I think most of the time folks just don't know how easy it'd be for them to publish a blog on the OTel site.

**Comment by avillela** [comment]
Yes, mostly aimed at end-user bloggers, but instead of getting them to write blog posts for the OTel blog, the idea is to get these folks to contribute content to the official docs. The comms SIG is really trying to prioritize docs content over blog content (see ["Documentation or Blog Post" in the blog contribution page](https://opentelemetry.io/docs/contributing/blog/#documentation-or-blog-post)), so how can we support them to make that happen ?
